### PR TITLE
Adjust metadata and printable to all languages

### PR DESCRIPTION
**Pull request by qwrtln**

### CODE REVIEW

File: make_printable.sh [hunk 1]
```diff
@@ -1,8 +1,15 @@
 #!/usr/bin/env bash
 
+LANGUAGE=$1
+en=1
+pl=2
+declare -p PATHS
+declare -a PATHS=( [1]="sections" [2]="sections/translated/pl" )
```
Comment (DarthGandalf) [inline make_printable.sh:7]: Why does it hardcode pl here?

**Review by DarthGandalf — APPROVED** [review]

File: make_printable.sh (hunk 1 above)
Comment (qwrtln) [reply make_printable.sh:7]: Let me add Spanish to better demonstrate this.

Comment (qwrtln) [reply make_printable.sh]: See the updated version. I wanted to choose the path with the input argument. I used bash associative array for this. I'm open for ideas.

Comment (DarthGandalf) [reply make_printable.sh]: See the suggestion. No need to list the languages here, and no need for this indirection through numbers

File: make_printable.sh [hunk 2]
```diff
@@ -1,8 +1,16 @@
 #!/usr/bin/env bash
 
+LANGUAGE=$1
+en=1
+pl=2
+es=3
+declare -p PATHS
+declare -a PATHS=( [1]="sections" [2]="sections/translated/pl" [3]="sections/translated/es" )
```
[the remark this answers is not in the document]
Comment (qwrtln) [reply make_printable.sh:8]: Yay, good idea!

File: make_printable.sh [hunk 3]
```diff
@@ -1,8 +1,20 @@
...
+en=1
+pl=2
+es=3
+declare -p PATHS
+if [[ ${LANGUAGE} == en ]]; then
+  SECTIONS=sections
+else
+  SECTIONS="sections/translated/${LANGUAGE}"
+fi
+SECTIONS=${PATHS[$LANGUAGE]}
```
Comment (DarthGandalf) [inline make_printable.sh:13]:
```suggestion
```

**Review by evilmint — APPROVED** [review]

**Review by luischos — APPROVED** [review]

**Review by Heegu-sama — APPROVED** [review]

**Pull request merged**